airlock/business_logic.py

```diff
@@ -674,6 +677,19 @@ def get_or_create_current_request(
 
         return self._create_release_request(workspace_name, user.username)
 
+    def get_requests_for_workspace(
+        self, workspace_name: str, user: User
+    ) -> list[ReleaseRequest]:
+        """Get all release requests in workspaces a user has access to."""
+
+        if not user.output_checker and workspace_name not in user.workspaces:
+            raise BusinessLogicLayer.RequestPermissionDenied(workspace_name)
```

Comment:
Elsewhere we use `self.RequestPermissionDenied`, and usually we provide a fuller error message than just the name of the workspace.

Reply:
yes, fair enough, we also do exactly this here https://github.com/opensafely-core/airlock/blob/main/airlock/business_logic.py#L673 but will change & change the other instance in a follow-up PR